README.md

```diff
@@ -141,34 +141,37 @@ Build jar and docker image:
 ```
 
 ### To run unit tests
-Spin up a local OpenDJ:
+#### Set up your environment
 ```
+#Spin up a local OpenDJ:
 sh docker/run-opendj.sh start
-```
-Spin up a local postgres:
-```
+#Spin up a local postgres:
 sh docker/run-postgres.sh start
-````
-
-
-Make sure your `SBT_OPTS` are set:
-```
+#Make sure your `SBT_OPTS` are set:
 export SBT_OPTS="-Dpostgres.host=localhost -Dpostgres.port=5432 -Ddirectory.url=ldap://localhost:3389 -Ddirectory.password=testtesttest"
 ```
 
-Run tests:
+#### Run tests in sbt
 ```
 sbt test
 ```
-Stop your local opendj:
+
+#### Run tests in intellij
+In `Run` -> `Edit Configurations...`, make sure the ScalaTest template has `VM Parameters` filled in with:
```

Comment:
I would emphasize that it's the _**template**_ that you're modifying here.  Rewording here slightly to maybe add a sub-heading or a bullut before these instructions to emphasize that it's the template you want to edit.  Or maybe an extra sentence describing how to navigate to the list of templates and find the right ScalaTest one to modify.